azurepipelines/adu-debian-arm32-build.yml

```diff
@@ -28,7 +28,8 @@ trigger:
   branches:
     include:
       - main
-      - release/*
+      # TODO(shiyipeng): Reactivate the pipeline after PM confirm on debian arm support
+      # - release/*
```

Comment:
remove commented lines.  Create task for follow up in future, add note in task as to what to add back into yml.
assign task to PM for now.

Reply:
Task 32092462: Confirm on Distro/Arch support breadth, added :smile: 

---
In reply to: [592797799](https://github.com/Azure/iot-hub-device-update/pull/13#discussion_r592797799) [](ancestors = 592797799)